File: app/models/episode.rb
class Episode < ActiveRecord::Base
  attr_accessible :long_description, :posted_time, :short_description, :title
  
  validates :title, :presence => true
  validates :short_description, :presence => true
  validates :long_description, :presence => true
  
  def posted
    posted_time ? posted_time <= Time.now : false
  end
  
  def asset_folder
    "/assets/episodes/#{id}"
  end
  
  def small_image_src
    "#{asset_folder}/small.jpg"
  end
  
  def large_image_src
    "#{asset_folder}/large.jpg"
  end
  
  def video_src
    "#{asset_folder}/video.webm"
  end
  
end

File: app/assets/javascripts/models/episode.js
Embersodes.Episode = Embersodes.Model.extend({
	title: DS.attr('string'),
	shortDescription: DS.attr('string'),
	longDescription: DS.attr('string'),
	postedTime: DS.attr('date'),
	smallImageSrc: DS.attr('string'),
	videoSrc: DS.attr('string')
});

/*Embersodes.episodes = [
	Embersodes.Episode.create({ id: 1, imageSrc: "assets/small1.jpg", title: "Introduction to MVC in Ember.js", shortDescription: "A guide to the bare essentials of Ember.js objects and views." }),
	Embersodes.Episode.create({ id: 2, imageSrc: "assets/small6.jpg", title: "Routing in Ember", shortDescription: "Build client-side navigation with the router built into Ember.js." })
]*/